docs/swagger.yaml

```diff
@@ -116,11 +111,24 @@ components:
           type: boolean
         path:
           type: object
-    Log:
-      type: object
-      properties:
-        log:
-          type: string
+    DeployLog:
+      type: array
+      items:
+        type: object
+        properties:
+          containerName:
+            type: string
+          logs:
+            type: array
+            items:
+              type: object
+              properties:
+                level:
+                  type: string
+                message:
+                  type: string
+                timestamp:
+                  type: string

```

Comment:
Só pra conferir: É timestamp (int) ou date (string)?
Se for date, pode adicionar um format e mudar o nome do campo para date:
```
type: string
format: date-time
```

Reply:
Coloquei como string pq tah vindo mais de um timestamp no log
2020-05-13T18:53:39.605125283Z 2020-05-13 18:53:39,600